src/base/bittorrent/dbresumedatastorage.cpp

```diff
@@ -109,42 +112,50 @@ namespace
         return QString::fromLatin1("CREATE TABLE %1 (%2)").arg(quoted(tableName), items.join(QLatin1Char(',')));
     }
 
-    QString makeInsertStatement(const QString &tableName, const QVector<Column> &columns)
+    std::pair<QString, QString> joinColumns(const QVector<Column> &columns)
```

Comment:
off topic, should we decide on one between `std::pair` and `QPair` in qbt codebase?
I would favor `std::pair`.

Reply:
> off topic, should we decide on one between `std::pair` and `QPair` in qbt codebase? I would favor `std::pair`.

I think we discussed similar issues (maybe in passing). I think we should use std types unless a similar Qt type has obvious advantages (for example, implicit sharing).